### ISSUE TITLE
@valibot & @valibot/to-json-schema version mismatch on NPM

### ISSUE THREAD

**Issue by hongkongkiwi** [fabian-hiller/valibot#928]
I can't seem to install @valibot/to-json-schema , it seems like there's a version mismatch between valibot and @valibot/to-json-schema on NPM.

After installing valibot I get version 1.0.0-beta.3 which looks fine.

Then when installing to-json-schema (using Deno), I get this error:
```
deno add npm:@valibot/to-json-schema
Add npm:@valibot/to-json-schema@1.0.0-beta.2
error: Could not find npm package 'valibot' matching '^1.0.0'.
```

**Comment by hongkongkiwi** [comment]
For others using Deno, here's a workaround in your deno.json:
```
"imports": {
  "valibot": "npm:valibot@^1.0.0-beta.3",
  "@valibot/to-json-schema": "https://raw.githubusercontent.com/fabian-hiller/valibot/refs/heads/main/packages/to-json-schema/src/index.ts"
}
```

**Comment by fabian-hiller** [comment]
Does it work when installing the @beta version of Valibot?

**Comment by fabian-hiller** [comment]
This is fixed in v1.0.0-beta.3 of `@valibot/to-json-schema`
